main.js

```diff
@@ -59,7 +76,57 @@ function getCurrentGutter() {
 	return getGridProperties().gutter;
 }
 
+/**
+ * This sets MediaQueryListeners on all the o-grid breakpoints
+ * and fires a `o-grid.layoutChange` event on layout change.
+ */
+function addBreakpointListeners() {
+	try {
+		// Create a map containing all breakpoints exposed via html:before
+		const breakpoints = new Map(Object.entries(getGridBreakpoints().layouts));
```

Comment:
We can't use `Object.entries` as a polyfill for it does not exist in https://polyfill.io as of yet. If you are already using this method in FT products please let me know and I will ensure to add it to https://polyfill.io.